WordPress/Classes/Utility/Analytics/WPAnalytics+Domains.swift

```diff
@@ -7,9 +7,8 @@ extension WPAnalytics {
         FeatureFlag.siteCreationDomainPurchasing.enabled && ABTest.siteCreationDomainPurchasing.isTreatmentVariation
     }
 
-    static func domainsProperties(for blog: Blog) -> [AnyHashable: Any] {
-        // For now we do not have the `siteCreation` route implemented so hardcoding `menu`
-        domainsProperties(usingCredit: blog.canRegisterDomainWithPaidPlan, origin: .menu)
+    static func domainsProperties(for blog: Blog, origin: DomainPurchaseWebViewViewOrigin? = .menu) -> [AnyHashable: Any] {
+        domainsProperties(usingCredit: blog.canRegisterDomainWithPaidPlan, origin: origin)
     }
```

Comment:
Added `origin` param with `.menu` as the default value to avoid breaking existing callers of this method.